database/src/migrations/20221005063930_api_delete_survey copy.ts

```diff
@@ -0,0 +1,67 @@
+import { Knex } from 'knex';
```

Comment:
The name of this file is a bit messed up, has `copy` in it at the end.

Reply:
please find adjusted.